scipy/integrate/_quad_vec.py

```diff
@@ -408,24 +431,28 @@ def quad_vec(f, a, b, epsabs=1e-200, epsrel=1e-8, norm='2', cache_size=100e6, li
     res = global_integral
     err = global_error + rounding_error
 
-    if full_output:
-        res_arr = np.asarray(res)
-        dummy = np.full(res_arr.shape, np.nan, dtype=res_arr.dtype)
-        integrals = np.array([interval_cache.get((z[1], z[2]), dummy)
-                                      for z in intervals], dtype=res_arr.dtype)
-        errors = np.array([-z[0] for z in intervals])
-        intervals = np.array([[z[1], z[2]] for z in intervals])
+    res_arr = np.asarray(res)
+    dummy = np.full(res_arr.shape, np.nan, dtype=res_arr.dtype)
+    integrals = np.array([interval_cache.get((z[1], z[2]), dummy)
+                                    for z in intervals], dtype=res_arr.dtype)
+    errors = np.array([-z[0] for z in intervals])
+    intervals = np.array([[z[1], z[2]] for z in intervals])
+    success = (ier == CONVERGED)
+    message = status_msg[ier]
 
+    if full_output:
         info = _Bunch(neval=neval,
```

Comment:
Let's add a note in the deprecation tracker than when `full_output` is removed, `_Bunch` can be removed.